### ISSUE TITLE
Send rare or longterm symptoms to PEI

### ISSUE THREAD

**Issue by skyerjoe** [corona-warn-app/cwa-wishlist#743]
## Feature description
Possibility to  send rare or longterm symptoms if u have or had an infection/ or  being vaxxinated to pei.

## Problem and motivation

Unknown/RARE symptoms could be gathered easily and sended to the pei for further research.

## Is this something you're interested in working on

---
Internal Tracking ID: [EXPOSUREAPP-11180](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11180)

**Comment by Jo-Achim** [comment]
For general understanding:
Vaccination side effects should be entered in the CWA and forwarded from the CWA to the PEI?

Wouldn't it be easier then to download the form directly from the [PEI website](https://www.pei.de/DE/arzneimittelsicherheit/pharmakovigilanz/meldeformulare-online-meldung/meldeformulare-online-meldung-node.html), fill it in and send it to the PEI?
Especially since the form requires some data that you might not want to enter in the CWA; for example personal data such as addresses, previous illnesses, medication, or even medical diagnoses, etc. Possibly plus medical documents.

Or does it just mean that there should be a link in the CWA to the relevant PEI registration forms?

**Comment by skyerjoe** [comment]
Hello @Jo-Achim  
Thx for reply 

The process is for the normal user too complicated if u only give an link to the pdf.

An  interface in cwa for it  with some questions to mark on will be more reliable.

According data, u could make it optional.

Best regards

**Comment by larswmh** [comment]
Thanks for your suggestion @skyerjoe. We have created an internal ticket for it and will raise this topic internally.
Internal Tracking ID: [EXPOSUREAPP-11180](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11180)

---
Corona-Warn-App Open Source Team

**Comment by Ein-Tim** [comment]
@larswmh @skyerjoe 

This seems like a duplicate of #229 to me, or?

**Comment by skyerjoe** [comment]
> @larswmh @skyerjoe
> 
> This seems like a duplicate of #229 to me, or?

#229 is only for side effects  related to Vaccinations or?

**Comment by Ein-Tim** [comment]
@skyerjoe 

Yes, I suggest you update your issue to be only for the long term effects of an infection.
